src/biogeochem/CNPhenologyMod.F90

```diff
@@ -3474,9 +3473,11 @@ subroutine CNLivewoodTurnover (num_soilp, filter_soilp, &
                      livestemn_to_retransn(p)  = 0
                   end if
                end if !use_matrixcn
+
             end if
+            livecrootn_to_retransn(p)  = ntovr - livecrootn_to_deadcrootn(p)
```

Comment:
not sure what case you're comparing to, @ekluzek but the wood CN changes I made may also generate differences here too?

Reply:
The thing that was wrong on the CN branch, was that this (the livecrootn_to_retransn assignment statement) was previously inside both the "if (woody)" and the "if ( lflex-CN )" if statements. It needs to be outside of the later, but inside the former according to the baseline code. The change that you worked on that came in ctsm1.0.dev109 was around this specific line, but didn't actually change that specific line of code.

When I compare the cn-matrix code to the baseline version of ctsm5.1.dev010 thie logic for this statement now looks correct on the matrix branch.